### PR TITLE
fix(api): setting content type on ccd upload

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport-internal#1730

### Description

- Setting the content type to `application/xml` on CCD file upload

### Testing

- Local
  - [x] Generated one from local env and got file info using s3 utils

### Release Plan
- [ ] Merge this

### CODE REVIEW

File: packages/api/src/external/cda/generate-ccd.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export async function generateCcd({
 
   return `<ClinicalDocument xmlns="urn:hl7-org:v3" xmlns:sdtc="urn:hl7-org:sdtc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" moodCode="EVN">
 	<realmCode code="US"/>
-	<typeId root="2.16.840.1.113883.1.3" extension="POCD_HD000040"/>
+  <typeId root="2.16.840.1.113883.1.3" extension="POCD_HD000040"/>
```
Comment (Goncharo) [inline packages/api/src/external/cda/generate-ccd.ts:35]: intentional to remove the spaces?

**Review by Goncharo — APPROVED** [review]
:shipit:

File: packages/api/src/external/cda/generate-ccd.ts (hunk 1 above)
Comment (RamilGaripov) [reply packages/api/src/external/cda/generate-ccd.ts:35]: No. Not sure why it's like that. In my IDE, it shows normally.. :/

**Comment by RamilGaripov** [comment]
Committed a change thru GH, and now unable to pass PR checks

**Pull request closed without merging**